web/__init__.py

```diff
@@ -80,13 +103,26 @@ def generate():
 
 @app.get("/")
 def app_root():
-    host = request.host.split(':')
-    requestPort = host[1] if len(host) > 1 else '80' # If there is no 2nd array entry, the request port is 80
-    return render_template(
-        "index.html",
-        requestPort=requestPort,
-        requestHost=host[0]
-    )
+    config = app.config["config"]
+    with config.open() as cfg:
+        user_agent = user_agent_parse(request.headers.get('User-Agent'))
+
+        host = request.host.split(':')
+        # If there is no 2nd array entry, the request port is 80
+        request_port = host[1] if len(host) > 1 else '80'
+        no_config = '<p>No printers found, please load your login config...</p>'
+        anker_config = str(web.config.config_show(
```

Comment:
The `anker_config` line should be a normal `if` statement. This is becoming messy.

That way we also get rid of `no_config` as a variable.

Reply:
Yeah, I prefer inline Ifs for variable setting but can move it to an if control block instead :)

Reply:
✅